.github/workflows/test.yml

```
@@ -1,35 +1,44 @@
     branches:
       - main
-      - develop
 
 jobs:
   validate:
-    name: Lint & Type Check
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    steps:
+      - name: Validate source branch
+        run: |
```

actionlint (1.7.10)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)